src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -1422,8 +1422,28 @@
      * stylesheetfile</a>.
      */
     @Parameter( property = "stylesheetfile" )
-   private String stylesheetfile;
-
+    private String stylesheetfile;
+    
+    /**
+     * Specifies the path of an additional HTML stylesheet file.
+     * @since 3.1.2
+     */
+    @Parameter
+    private String addStylesheet;
+    
+    /**
+     * Specifies the path of an additional HTML stylesheet file.
+     * Example:
+     * <pre>
+     *     &lt;addStylesheets&gt;

```

Comment:
are tag names like this typically nouns or verbs?

Reply:
if we would control the name of the parameter, most likely yet. But this one matches the javadoc tool parameter